obs-packaging/create-repo-branch.sh

```diff
@@ -0,0 +1,145 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+[ -z "${DEBUG}" ] || set -o xtrace
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+script_dir=$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)
+script_name="$(basename "${BASH_SOURCE[0]}")"
+
+# shellcheck source=./../scripts/lib.sh
+source "${script_dir}/../scripts/lib.sh"
+
+# shellcheck source=scripts/obs-docker.sh
+source "${script_dir}/scripts/obs-docker.sh"
+
+readonly home_project="home:katacontainers"
+readonly template_pkg="kata-pkg-template"
+arch_target=$(uname -m)
+
+projects=(
+	qemu-lite
```

Comment:
@jodh-intel @jcvenegas : Can qemu-lite packaging be made arch specific as ppc64le would not be needing it. qemu-vanilla should be good enough for Kata Containers to work on ppc64le?